frontend/src/containers/QuestionNumberPage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../styles/App.scss';
+
+import questionpagedesign from '../styles/QuestionNumberPage.module.scss';
+function QuestionNumberPage() {
+    return (
+        <div>
+            <div className={questionpagedesign.centercontainer }>
+                <div className={questionpagedesign.toporange }>
+
+                </div>
```

Comment:
You can use <div /> instead of <div></div> so your code is more organized and concise